docs/guides/file/file.md

```
@@ -26,6 +27,8 @@ RAGFlow file management lets you build your own file system through nested folde
 
 After clicking **Add File** in the upper-right corner of the page, select **Upload File** to open the upload window. The upload control contains two tabs: **File** and **Folder**. You can select one or more files, or upload an entire folder.
 
+![](https://raw.githubusercontent.com/infiniflow/ragflow-docs/main/images/upload_files.jpg)
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)